src/client/styles/components/_ContentBanner.scss

```diff
@@ -201,16 +225,83 @@
         @include displayVisuallyHidden;
         color: #FFF;
         width: auto; // Override displayVisuallyHidden width rule
-        margin: 25px 0 0;
+        margin: 5px 0 0;
+
+        p {
+          width: 65%;
+        }
       }
 
       @include min-screen($tablet-landscape + 1px) {
-        max-width: 100%;
+        max-width: 60%;
+        p {
+          width: 100%;
+        }
       }
 
       @include min-screen($desktop) {
         margin-top: 30px;
-        max-width: 420px;
+        max-width: 300px;
+      }
+    }
+
+    .seeMoreLink {
+      top: 75%;
+      right: 15px;
+
+      @include Kievit-Medium;
+      border: 2px solid $WHITE;
+      border-radius: 22px;
+      color: #000;
+      background: $WHITE;
+      text-decoration: none;
+      position: absolute;
+      height: 35px;
+      padding: 0 4px;
+      text-align: center;
+      vertical-align: middle;
+      font-size: 0.8rem;
+
+      @include min-screen(380px) {
+        top: 80%;
+      }
+
+      @include min-screen(480px) {
+        top: 85%;
+      }
+
+      @include min-screen(600px) {
+        top: 90%;
+      }
+
+      @include min-screen($tablet-portrait) {
+        border: 2px solid #333;
```

Comment:
$PRIMARY_COLOR

Reply:
`$PRIMARY_COLOR` is red, unless if you mean something else?:

![screen shot 2018-10-12 at 10 59 10 am](https://user-images.githubusercontent.com/1280564/46877058-dcda7080-ce0d-11e8-8299-4f507fa244cc.png)

Reply:
Oh I meant the arrow and the border being red in the mobile/tablet as per mock. Thanks for checking!